<launch>
  <arg name="camera_topic" default="/zed/left/image_raw" />
  <arg name="weights_path" default="$(find yolov10_cone_detection)/weights/aambest.pt" />
  <arg name="conf_threshold" default="0.65" />

  <node pkg="yolov10_cone_detection"
        type="yolo_cone_detection_node.py"
        name="yolo_cone_detection_node"
        output="screen">
    <param name="camera_topic" value="$(arg camera_topic)"/>
    <param name="weights_path" value="$(arg weights_path)"/>
    <param name="conf_threshold" value="$(arg conf_threshold)"/>
  </node>
</launch>
